oe_corporate_blocks.post_update.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Corporate blocks Demo post updates.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Delete the config for Site Switcher.
+ */
+function oe_corporate_blocks_post_update_00001_delete_config_of_site_switcher() {
```

Comment:
We should use the [new convention](https://github.com/openeuropa/documentation/blob/master/docs/architecture/third-party/drupal/module-update-policy.md) here.

Reply:
done